routes/patient.js

```diff
@@ -1,11 +1,10 @@
 const express = require("express");
 const router = express.Router();
-const patient = require("../models/patient");
 const Patient = require("../controllers/patientController");
-const Smartbed = require("../models/smartbed");
 
 router.get("/", Patient.getPatients);
 router.get("/:id", Patient.getPatientById);
+router.get("/:nric/nric", Patient.getPatientByNric);
```

Comment:
think it's more intuitive to be calling the query type first then the query itself so /nric/:nric